packages/react/dismissable-layer/src/DismissableLayer.tsx

```diff
@@ -293,7 +297,10 @@ function usePointerDownOutside(onPointerDownOutside?: (event: PointerDownOutside
  * Listens for when focus happens outside a react subtree.
  * Returns props to pass to the root (node) of the subtree we want to check.
  */
-function useFocusOutside(onFocusOutside?: (event: FocusOutsideEvent) => void) {
+function useFocusOutside(
+  ownerDocument: Document,
+  onFocusOutside?: (event: FocusOutsideEvent) => void
+) {
```

Comment:
For consistency, could we follow the same API as you did in `onEscapeKeydown` where the `ownerDocument` comes second and default to `document`?

Reply:
Done.